cmd/virtual-kubelet/internal/commands/root/root.go

```diff
@@ -165,6 +155,17 @@ func runRootCommand(ctx context.Context, s *provider.Store, c Opts) error {
 			log.G(ctx).Debug("created new node")
 			return nil
 		}),
+	}
+	if c.EnableNodeLease {
+		leaseClient := nodeutil.NodeLeaseV1Client(client)
+		// 40 seconds is the default lease time in upstream kubelet
+		additionalOptions = append(additionalOptions, node.WithNodeEnableLeaseV1(leaseClient, 40))
```

Comment:
Interesting. The docs say 10s but the code does indeed default to 40s https://github.com/kubernetes/kubernetes/blob/8b8de0316881a2a8958c2f327d6c3949ec5fe355/pkg/kubelet/apis/config/v1beta1/defaults.go#L129

Reply:
For easier reading/change in the future, can we please set this as a const in the beginning of the file?

Reply:
I moved this to a const